### ISSUE TITLE
Saved `field` parameter is now invalid.

### ISSUE THREAD

**Issue by themontem** [elastic/beats#6489]
After loading index template and dashboard and creating Index pattern for `packetbeat-*`. Getting the issue `Saved field parameter is now invalid. Please select a new field.`

![image](https://user-images.githubusercontent.com/14853016/36824900-bfa934e0-1d2a-11e8-8ce1-d2fab3142c51.png)

- **Version**:
ElasticSearch: `Version: 6.2.1, Build: 7299dc3/2018-02-07T19:34:26.990113Z, JVM: 1.8.0_111`
Kibana: `6.2.2`
Packetbeat: `Master Branch, Last Commit Hash: 911a1ab26dd8cc64ec4fb1f3fa228512066a3ef2`

- **Operating System:** `macOS 10.13.2, Darwin Kernel Version 17.3.0`
- **Steps to Reproduce:**
   - Compile a Packetbeat by using the Makefile.
   - Load index template and Dashboard
`sudo ./packetbeat setup -strict.perms=false -e --path.home="_meta/" -c "../packetbeat.yml"`
   - Start Packetbeat
`sudo ./packetbeat -strict.perms=false -e`

**Comment by manthansharma** [comment]
Kibana working fine with `packetbeat version 6.2.2 (amd64), libbeat 6.2.2`

**Comment by oaten-cracker** [comment]
Hey team, 

not sure if this is the right ticket to attach to, but I got the same issue with Filebeat and Kibana. Almost all visualisations except some, loose their mapped field. One solution, which is relativly time consuming is to set the "field" in the visualisations oneself.

Versions: Debian Stretch (4.13); Elasticsearch 6.2.4; Logstash 6.2.4; Kibana 6.2.4; Filebeat 6.2.4

![system_dashboard_bug1](https://user-images.githubusercontent.com/14074026/40644601-f94657a2-6323-11e8-8f9d-0090a14e0d85.png)

![system_dashboard_bug2](https://user-images.githubusercontent.com/14074026/40644624-06eb3bf2-6324-11e8-95f9-07ca7c6dcd21.png)

**Comment by mehdih29** [comment]
+1

**Comment by ruflin** [comment]
I finally got to investigate it and I can reproduce it. This discuss thread might have some interesting pointers from @Stacey-Gammon https://discuss.elastic.co/t/saved-field-parameter-is-now-invalid-please-select-a-new-field-visualize-field-is-a-required-parameter/70034/22 and here are the related Github issues:

* https://github.com/elastic/kibana/issues/9466 
* https://github.com/elastic/elasticsearch/issues/22438

**Comment by ruflin** [comment]
I just tried to reproduce this again. This time with `packetbeat 7.0, es/kb 7.0` and `packetbeat 7.0, es/kb 6.3` but in both cases I didn't see the error anymore. As I used the same setup before I'm not sure what changed :-(

**Comment by oaten-cracker** [comment]
Can this happen due to not enabling (File)Beat to sending logs directly to Elasticsearch?
Im currently using the example Logstash filter, to parse the modules output to Elasticsearch. To actually install the dashboards, I installed Filebeat on the ELK server and run the setup routine from there (elasticsearch and kibana in my installation are only listening on localhost).
After I changed the index id for the visualisations, I had to manually change the field mappings, because the actual values where not in the mapped 'field', but in 'field.keyword'.
